bpf/lib/identity.h

```diff
@@ -6,6 +6,21 @@
 
 #include "dbg.h"
 
+#define LOCAL_CLUSTER_IDENTITY_RANGE_START 1
+#define LOCAL_CLUSTER_IDENTITY_RANGE_END   ((1 << 16) - 1)
+#define MULTI_CLUSTER_IDENTITY_RANGE_START (1 << 16)
+#define MULTI_CLUSTER_IDENTITY_RANGE_END   ((1 << 24) - 1)
+#define CIDR_IDENTITY_RANGE_START          (1 << 24)
+#define CIDR_IDENTITY_RANGE_END            ((1 << 24) + (1 << 16) - 1)
```

Comment:
I'm guessing these are defined in the agent however, so maybe worth passing them via `node_config.h` to ensure we always use the same values in the agent and in the datapath.

Reply:
As a start, as far as the agent is concerned, CIDR identities are just "local" identities, so we need to look into `pkg/identity/cache/local.go`.

The `localIdentityCache` is initialized as follow: https://github.com/cilium/cilium/blob/e3b4464f3c2d35e1c3410406138682a2f07ab44c/pkg/identity/cache/allocator.go#L268

where `1` and `0xFFFFFF` are the min/max values.

Then, when we allocate a ~CIDR~ local identity, we mask the value returned by the allocator with `LocalIdentityFlag`: https://github.com/cilium/cilium/blob/e3b4464f3c2d35e1c3410406138682a2f07ab44c/pkg/identity/cache/local.go#L61

which is set to: https://github.com/cilium/cilium/blob/e3b4464f3c2d35e1c3410406138682a2f07ab44c/pkg/identity/numericidentity.go#L25

So we could define those min/max values as `LocalIdentity{Min,Max}ID` (although that would be a bit confusing as they are not the actual min and max IDs) and then pass `LocalIdentity{Min,Max}ID | LocalIdentityFlag` to the datapath as constants.

This also confirms that the range map in the docs is incorrect (may be the same case for the multicluster range)